File: Phasegate/DayOne/Level 1/java/TEST2/TestTaskTwo.java
import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.*;
public class TestTaskOne{
    @Test
    public void testReplaceNonPerfectSquares(){
        int[] number   = {4, 7, 9, 10, 49, 6};
        int[] expected = {9, -1, 9, -1, 49, -1};
        int[] actual   = TaskOne.replaceNonPerfectSquares(number);
        assertArrayEquals(expected, actual);
    }
}

File: Phasegate/DayOne/Level 1/python/TEST2/test_task_two.py
from unittest import TestCase
import task_two
class task_two_test(TestCase):
    def test_that_non_perfect_squares_are_replaced(self):
        numbers  = [4, 7, 9, 10, 16, 18]
        actual   = task_two.replace_non_perfect_squares(numbers)
        expected = [4, -1, 9, -1, 16, -1]
        self.assertEqual(actual, expected)
